src/main/uni_hid_device_vendors.h

```diff
@@ -602,6 +602,7 @@ static const uni_controller_description_t arrControllers[] = {
         // 8-Bitdo controllers
         { MAKE_CONTROLLER_ID(0x2820, 0x0009), CONTROLLER_TYPE_8BitdoController, NULL},  // 8Bitdo NES30 Gamepro (Bluepad32)
         { MAKE_CONTROLLER_ID(0x2dc8, 0x6101), CONTROLLER_TYPE_8BitdoController, NULL},  // 8Bitdo SN30 pro (Bluepad32)
+	{ MAKE_CONTROLLER_ID(0x2dc8, 0x3230), CONTROLLER_TYPE_8BitdoController, NULL},  // 8Bitdo zero 2 (Bluepad32) // added 25-01-2021 from vcorp
```

Comment:
could you remove the date and author?

The code should be clean, easy to read. Imagine if for each line that is added, a date and author is appended to it.
Date and author is registered in git. And also the AUTHORS file will give credit to you.

The same for the Sony line.

Reply:
Cleaned all with a new branch